src/main/java/com/kodilla/ecommercee/dto/PaymentDto.java

```diff
@@ -0,0 +1,25 @@
+package com.kodilla.ecommercee.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.math.BigDecimal;
+
+@Getter
+@Setter
```

Comment:
Można też @ Data zamiast @ Getter i @ Setter ;)

Reply:
W pozostałych jest @Getter i @Setter, więc tutaj też użyłemm tak. W oddzielnym zadaniu mozna to wszystko zmienic